test/visual-diff/visual-tests

```diff
@@ -107,6 +107,27 @@
    *   // that a user would. (e.g., click on links, swipe gallery images.)
    *   "enable_percy_javascript": true,
    *
+   *   // [optional, EXPERIMENTAL] Path of JavaScript file relative to webserver
+   *   // root that exports a dictionary of (async) test functions keyed by a
+   *   // short descriptive name. Each test function is passed two arguments: a
+   *   // reference to the Puppeteer page, and the full name of the test (page
+   *   // name + test name). e.g., the JS file can be:
+   *   //   module.exports = {
+   *   //     'tap red button': async (page, name) => {
```

Comment:
is the key here sort of just a description? it does nothing as far as execution correct?

Reply:
It's a description, but it also sets the name of the snapshot in Percy - and that determines what snapshot any future PR will be compared against

e.g., in the example test that I added, I named the test `tap "see more" button`, and in Percy the name of the snapshot is `AMP List and Mustache (tap "see more" button)`
https://percy.io/ampproject/amphtml/builds/1137857